src/vso-task-lib/lib/vsotask.js

```diff
@@ -223,3 +256,8 @@ var _filter = function (pattern, options) {
     return minimatch.filter(pattern, options);
 }
 exports.filter = _filter;
+
+var _findFiles = function (pattern) {
+    return glob.sync(pattern);
```

Comment:
I was trying to avoid duplicating the code for find and filter in all the tasks. Will remove the glob.sync and use find and filter methods in the task implementation